strongbox-event-api/src/main/java/org/carlspring/strongbox/event/artifact/ArtifactEventTypeEnum.java

```diff
@@ -33,11 +33,6 @@
      */
     EVENT_ARTIFACT_CHECKSUM_UPDATED(5),
 
-    /**
-     * Occurs when an artifact metadata file has been uploaded.
-     */
-    EVENT_ARTIFACT_METADATA_UPLOADED(6),
```

Comment:
What's the reason to remove this?

Reply:
because it not used, and many other events not used, almost all of them
why do we need unused events?